tests/transforms/test_geometry.py

```diff
@@ -45,4 +45,13 @@ def test_ccw_both_cw(self):
         line1 = [(0, 1), (1, 1)]
         line2 = [(1, 0), (0, 0)]
         result = geo.get_ccw_vertices_from_two_lines(line1, line2)
-        self.assertEqual(result, [(0, 1), (0, 0), (1, 0), (1, 1), (0, 1)])
\ No newline at end of file
+        self.assertEqual(result, [(0, 1), (0, 0), (1, 0), (1, 1), (0, 1)])
+
+    def test_ccw_boundary_check(self):
```

Comment:
change the name of the test to something like `test_prune_two_lines`

Reply:
changed